vendor.conf

```diff
@@ -6,21 +6,21 @@ github.com/davecgh/go-spew v1.1.0
 github.com/pmezard/go-difflib v1.0.0
 golang.org/x/sys 9aade4d3a3b7e6d876cd3823ad20ec45fc035402 # update to 7ddbeae9ae08c6a06a59597f0c9edbc5ff2444ce wiht pkg/signal change
 
-github.com/containerd/containerd v1.0.0
+github.com/containerd/containerd 90553efdef0678b2609aed74926a487f8ff58d1a
```

Comment:
v1.0.1-rc0 for consistency?

Reply:
This is not the same commit but master. The client has different stability properties than the API in containerd.